### DISCUSSION TITLE
Interface webcamiod to a video streamed to the PC

### DISCUSSION THREAD

**Discussion by xianghui-renesas** [webcamoid/webcamoid#588] [General]
Hi hello everyone,
I have a video streamed from a PCB board to the PC over ethernet port. 
Would it be possible to display this video streaming using webcamiod?
Appreciate any comments you may have.
thanks!

**Comment by hipersayanX** [comment]
Sure, add the source as a localhost URL.

![Screenshot_20230428_123635](https://user-images.githubusercontent.com/1249885/235192387-d1cc4314-98a4-495e-bdbd-2386f42e9dae.png)

**Reply by xianghui-renesas** [reply]
Hi hipersayanX, thanks for your response! Does webcamiod needs to know the format of the video? How is this information acquired by webcamiod? thanks!

**Reply by hipersayanX** [reply]
Any format supported by FFmpeg or GStreamer is ok (webm, mp4, mkv, avi, mjpg, etc.), but it must be a raw link to the video.
